src/raftstore/store/config.rs

```diff
@@ -152,6 +152,14 @@ impl Config {
     }
 
     pub fn validate(&self) -> Result<()> {
+        if self.raft_heartbeat_ticks == 0 {
+            return Err(box_err!("heartbeat tick must greater than 0"));
+        }
+
+        if self.raft_election_timeout_ticks <= self.raft_heartbeat_ticks {
```

Comment:
This is just a copy-paste of https://github.com/pingcap/tikv/blob/master/src/raft/raft.rs#L138